packages/react-components/react-switch/src/components/Switch/useSwitchStyles.ts

```diff
@@ -22,175 +22,173 @@ const trackHeight = 20;
 const trackWidth = 40;
 const thumbSize = trackHeight - spaceBetweenThumbAndTrack;
 
-const useRootStyles = makeStyles({
-  base: {
-    alignItems: 'flex-start',
-    boxSizing: 'border-box',
-    display: 'inline-flex',
-    position: 'relative',
+const useRootBaseClassName = makeResetStyles({
+  alignItems: 'flex-start',
+  boxSizing: 'border-box',
+  display: 'inline-flex',
+  position: 'relative',
 
-    ...createFocusOutlineStyle({ style: {}, selector: 'focus-within' }),
-  },
+  ...createFocusOutlineStyle({ style: {}, selector: 'focus-within' }),
+});
 
+const useRootStyles = makeStyles({
   vertical: {
     flexDirection: 'column',
   },
 });
 
-const useIndicatorStyles = makeStyles({
-  base: {
-    ...shorthands.borderRadius(tokens.borderRadiusCircular),
-    ...shorthands.borderStyle('solid'),
-    ...shorthands.borderWidth('1px'),
-    lineHeight: 0,
-    boxSizing: 'border-box',
-    fill: 'currentColor',
-    flexShrink: 0,
-    fontSize: `${thumbSize}px`,
-    height: `${trackHeight}px`,
-    ...shorthands.margin(tokens.spacingVerticalS, tokens.spacingHorizontalS),
-    pointerEvents: 'none',
+const useIndicatorBaseClassName = makeResetStyles({
+  borderRadius: tokens.borderRadiusCircular,
+  borderStyle: 'solid',
+  borderWidth: '1px',
+  lineHeight: 0,
+  boxSizing: 'border-box',
+  fill: 'currentColor',
+  flexShrink: 0,
+  fontSize: `${thumbSize}px`,
+  height: `${trackHeight}px`,
+  margin: tokens.spacingVerticalS + ' ' + tokens.spacingHorizontalS,
+  pointerEvents: 'none',
+  transitionDuration: tokens.durationNormal,
+  transitionTimingFunction: tokens.curveEasyEase,
+  transitionProperty: 'background, border, color',
+  width: `${trackWidth}px`,
+
+  '@media screen and (prefers-reduced-motion: reduce)': {
+    transitionDuration: '0.01ms',
```

Comment:
This value seems like a potential candidate for a token.

Reply:
Probably, but there's no token for it at the moment as far as I can tell.